src/status_im2/contexts/shell/activity_center/view.cljs

```diff
@@ -53,26 +54,30 @@
                                            :override-theme :dark})])
                           (rf/dispatch [:hide-bottom-sheet]))}]]]))
 
+(def empty-tab-description
+  {types/no-type              (i18n/label :t/empty-notifications-all-tab)
+   types/admin                (i18n/label :t/empty-notifications-admin-tab)
+   types/mention              (i18n/label :t/empty-notifications-mentions-tab)
+   types/reply                (i18n/label :t/empty-notifications-replies-tab)
+   types/contact-request      (i18n/label :t/empty-notifications-contact-requests-tab)
+   types/contact-verification (i18n/label :t/empty-notifications-identity-verification-tab)
+   types/tx                   (i18n/label :t/empty-notifications-transactions-tab)
+   types/membership           (i18n/label :t/empty-notifications-membership-tab)
+   types/system               (i18n/label :t/empty-notifications-system-tab)})
+
 (defn empty-tab
```

Comment:
nit: this file is getting massive. @smohamedjavid, @mmilad75 maybe we should move all of the empty tab pieces to a separate file?

Reply:
Yes, we should keep the main view simple. We can move the `empty-tab`, `header`, `tabs`, `options-bottom-sheet-content`, ... etc to respective files. These notification types or files will grow in future.

Reply:
Great point 👍

Reply:
Done